sdk/identity/azure-identity/azure/identity/_credentials/environment.py

```diff
@@ -129,6 +131,7 @@ def get_token(self, *scopes: str, **kwargs: Any) -> AccessToken:
             For more information about scopes, see
             https://learn.microsoft.com/azure/active-directory/develop/scopes-oidc.
         :keyword str tenant_id: optional tenant to include in the token request.
+        :keyword str claims: not supported by this credential.
```

Comment:
Since this uses either a user or service principal credential underneath, i think we can add a claims docstring here.

```suggestion
        :keyword str claims: additional claims required in the token, such as those returned in a resource provider's
            claims challenge following an authorization failure.
```